src/base/projects/Widget.svelte

```diff
@@ -92,14 +92,14 @@
     display: flex;
     justify-content: space-between;
   }
-  article .id .urn {
+  article .id .radid {
```

Comment:
Would be nice if we'd stick to one way of naming this, so either `.radid` or `.radicle-id` everywhere.

Reply:
Hmm can we use `.rid`? Would match `RID`s in the doc, ie. repository id. Otherwise if it's for a user id, I'd use `public-key` I guess if it had to be non-ambiguous.

Reply:
replying for [this](https://github.com/radicle-dev/radicle-interface/pull/515#discussion_r1046813887) and [this](https://github.com/radicle-dev/radicle-interface/pull/515#discussion_r1046816859) here as well rudolfs. 

we can use `rid` in places where `id` is reserved but since we want to keep models 1-to-1 to api, using `rid` would change that, and I prefer sticking to `.id`.